tools/build_steps.sh

```diff
@@ -181,11 +181,14 @@ EOF
             export DYLD_LIBRARY_PATH=/usr/local/lib:$DYLD_LIBRARY_PATH
             ;;
         *-s390x)
+            # The TargetList.txt has only ZARCH_GENERIC, Z13, Z14. Not worth
+            # messing with dynamic lists and targets.
             local bitness=64
```

Comment:
```suggestion
            # messing with dynamic lists.
            local bitness=64
            local target="ZARCH_GENERIC"
```
see https://github.com/OpenMathLib/OpenBLAS/blob/f6df9bebbb4259aa61ab5634c0f1269fb152cc0e/README.md?plain=1#L271-L273